This broke one webkitpy test:

```
[319/2495] webkitpy.common.checkout.checkout_unittest.CheckoutTest.test_apply_patch erred:
  Traceback (most recent call last):
    File "/Volumes/Data/worker/WebKitPy-Tests-EWS/build/Tools/Scripts/webkitpy/common/checkout/checkout_unittest.py", line 373, in test_apply_patch
      checkout.apply_patch(mock_patch)
    File "/Volumes/Data/worker/WebKitPy-Tests-EWS/build/Tools/Scripts/webkitpy/common/checkout/checkout.py", line 173, in apply_patch
      num_commits = len(self.COMMIT_SUBJECT_RE.findall(encoded_patch))
  TypeError: cannot use a string pattern on a bytes-like object
```

--- a/Tools/Scripts/webkitpy/common/checkout/checkout.py
+++ b/Tools/Scripts/webkitpy/common/checkout/checkout.py
@@ -49,8 +49,8 @@ if sys.version_info > (3, 0):
 # FIXME: Move a bunch of ChangeLog-specific processing from SCM to this object.
 # NOTE: All paths returned from this class should be absolute.
 class Checkout(object):
-    COMMIT_SUBJECT_RE = re.compile(b'Subject: \[PATCH ?(\d+\/\d+)?] (.+)')
-    FILTER_BRANCH_PROGRAM = '''import re
+    COMMIT_SUBJECT_RE = re.compile(br'Subject: \[PATCH ?(\d+\/\d+)?] (.+)')
+    FILTER_BRANCH_PROGRAM = r'''import re
 import sys
 
 lines = [l for l in sys.stdin]

